spring-cloud-skipper-server-core/src/test/java/org/springframework/cloud/skipper/server/statemachine/StateMachineTests.java

```diff
@@ -134,32 +133,32 @@ public void testSimpleInstallShouldNotError() throws Exception {
 		Mockito.when(releaseService.install(any(), any())).thenReturn(new Release());
 
 		Message<SkipperEvents> message = MessageBuilder
-			.withPayload(SkipperEvents.INSTALL)
-			.setHeader(SkipperEventHeaders.PACKAGE_METADATA, new PackageMetadata())
-			.setHeader(SkipperEventHeaders.INSTALL_PROPERTIES, new InstallProperties())
-			.setHeader(SkipperEventHeaders.VERSION, 1)
-			.build();
+				.withPayload(SkipperEvents.INSTALL)
```

Comment:
Is there a code formatter update we can do to maintain the same format (especially the statemachine related builder code)

Reply:
In a same way than reactor stuff, these are manually formatted for things to stay readable.

Reply:
yea, I got a bit nervous when I hit reformat there, I prob needed to just optimize the imports.  There is an option I believe to turn off the code formatter for certain sections (we do this in the shell code).  I'll look to revert to the old formatted code and add those conditional code formatting guards.

Reply:
Can't seem to get `//CHECKSTYLE:OFF` working so just reverting back to the manually formatted code.